pkg/thing/interactors/update_schema.go

```diff
@@ -7,12 +7,10 @@ import (
 	"github.com/go-playground/validator"
 )
 
-// ErrInvalidSchema represents the error when the schema has a invalid format
-type ErrInvalidSchema struct{}
-
-func (eis *ErrInvalidSchema) Error() string {
-	return "Thing's schema is invalid"
-}
+var (
+	// ErrSchemaInvalid is returned for invalid schema formats.
+	ErrSchemaInvalid = fmt.Errorf("invalid schema")
```

Comment:
`fmt.Errorf` should be used only when it's necessary to fill some placeholders with additional data. Please, change to `errors.New()`.

Reply:
Done :smile: